xbmc/utils/ScreenshotAML.cpp

```diff
@@ -18,8 +18,6 @@
  *
  */
 
-#include "system.h"
```

Comment:
you need 

```
#include <string.h>
#include <unistd.h>
```

for mempcy() and pread(). those were previously included via PlatformDefs.h

Reply:
added